tests/basics/list_slice_3arg.py

```diff
@@ -26,3 +26,10 @@
 print(x[-1:-2:-1])
 print(x[-1:-11:-1])
 print(x[-10:-11:-1])
+
+print([][::-1])
+print([1][::-1])
+print([][:-20:-1])
+print([1][:-20:-1])
+print([][-20::-1])
+print([1][-20::-1])
```

Comment:
It might be a good idea to add a test for something like `print(x[:-15:-1])` (i.e. with a stop whose absolute value is greater than `len(x)`). On my PyBoard right now, this example (through -15) doesn't crash, but does exhibit reading outside of the list.